contributing.md

```diff
@@ -0,0 +1,164 @@
+
+<img src = "./docs/_images/jira-ticket.png" alt="JIRA ticket" width = 30% height = 30% />
+
+- Select the **open a JIRA ticket** link in the Contact Us section on the bottom of the page. This action opens the [Jira issue tracker](https://jira.percona.com/projects/PSQLADM/issues) for the doc project.
+
+- Log in (create a Jira account if you don't have one) and select **Create** to open the Jira form.
+  
+
+<img src = "./docs/_images/create-issue.png" alt="Complete the Jira form" width = 30% height = 30% />
+
+- In the following fields, describe the issue:
+
+  - In the Summary, provide a brief description of the issue.
+
+  - In the Description, provide more information about the issue. If needed, add a Steps To Reproduce section and information about your environment (version number, your operating system, etc.). Be detailed. 
+  
+  - Select **CREATE** to create the ticket.
+
+## Contribute to documentation yourself
+
+Using either the [Edit documentation online with GitHub](#edit-documentation-online-with-github) method or the [Edit documentation locally](#edit-documentation-locally) method to make changes to the documentation and create a pull request. 
+ 
+
+### What you should know
+
+Most of the document is in plain text, but you may use [Markdown] (https://www.markdownguide.org/) to add syntax elements (notes, tables, and so on) to the documentation. 
+
+### What happens after you create the pull request
+
+A core developer reviews your request and either comments or approves the request. A core developer merges the approved request to the **main** branch usually when a newer version of the product is released.
+
+!!! note
+
+    We appreciate your work but the pull request may be redone to meet internal requirements.
+
+### Edit documentation online with GitHub
+
+In the upper right corner, select the pencil icon to open the source file in the GitHub editor. In this editor, you can make the changes, view the changes in the Preview tab, and create a pull request. The source `.md` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page using the Markdown syntax.
+
+You should review your changes on the **Preview** tab.
+
+Commit your changes to a pull request.
+
+In the **Commit changes** section, add a summary (72 characters or less) of what was changed
+  
+Then select the **Create a new branch for this commit and start a pull request** option. Accept the name for the branch and then Select **Commit changes**.
+
+GitHub creates a branch and a commit for your changes. The operation loads a page that shows the following:
+
+-  a base branch where you offer your changes
+  
+- your commit message
+  
+- a visual representation of your changes against the original page 
+
+You can make a review the page and click **Create pull request**.
```

Comment:
You can make a review the page and click **Create pull request**.  -->. You can review the page and click **Create pull request**.